src/components/structures/GroupView.js

```diff
@@ -43,15 +43,15 @@ import {mediaFromMxc} from "../../customisations/Media";
 import {replaceableComponent} from "../../utils/replaceableComponent";
 
 const LONG_DESC_PLACEHOLDER = _td(
-`<h1>HTML for your community's page</h1>
-<p>
-    Use the long description to introduce new members to the community, or distribute
-    some important <a href="foo">links</a>
-</p>
-<p>
-    You can even add images with Matrix URLs <img src="mxc://url" />
-</p>
-`);
+    "<h1>HTML for your community's page</h1>" +
```

Comment:
Hmm, this change seems to have altered the English string, which means it will need to be retranslated... I'd say let's revert this change, since I would expect this UI to go away soonish when Spaces takes over, and so there's not much value in having a string for Communities go through translation again.